esp/bindings/http/platform/httptransport.ipp

```diff
@@ -446,7 +447,7 @@ class CFlushingHttpResponseBuffer : public CInterfaceOf<IXmlStreamFlusher>
 {
     CHttpResponse* response = nullptr;
     const size32_t responseFlushThreshold = defaultResponseFlushThresholdBytes;
-    const size32_t closingThreshold = 1; //When closing, no flushing is needed if responseBuffer.length() < closingThreshold.
+    const size32_t closingThreshold = closingResponseFlushThresholdBytes; //When closing, no flushing is needed if responseBuffer.length() < closingThreshold.
```

Comment:
this variable seems superfluous, let's remove it and use closingResponseFlushThresholdBytes directly in the ternary test on line 479
https://github.com/hpcc-systems/HPCC-Platform/blob/72515d346df4f7fee1609a289dcf7fbdfcc70d2f/esp/bindings/http/platform/httptransport.ipp#L479

Reply:
ok